src/js/navButtons.js

```diff
@@ -163,4 +163,34 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         that.applier.change("currentPanelNum", newValue);
     };
 
+    // A grade component for integrating the first discovery tool and the preferences server.
+    // It overrides some definitions on "gpii.firstDiscovery.navButtons"
+    fluid.defaults("gpii.firstDiscovery.navButtons.prefsServerIntegration", {
+        gradeNames: ["fluid.modelComponent"],
```

Comment:
As discussed, since ```gpii.firstDiscovery.navButtons.prefsServerIntegration``` is used as a add-on grade for navButtons instead of replacing the type of navButtons, will not add ```gpii.firstDiscovery.navButtons``` as a base grade.